src/main/scala-sbt-1.0/scalafix/sbt/ScalafixTestkitPlugin.scala

```diff
@@ -13,38 +13,42 @@ object ScalafixTestkitPlugin extends AutoPlugin {
   override def requires: Plugins = JvmPlugin
 
   object autoImport {
-    case class InputAxis(scalaVersion: String) extends VirtualAxis.WeakAxis {
+    case class TestkitTargetAxis(scalaVersion: String)
```

Comment:
not very happy about that name either... suggestions welcome!

Reply:
It's already better! Don't have better propositions